adafruit_lps2x.py

```diff
@@ -139,31 +146,36 @@ class LPS2X:  # pylint: disable=too-many-instance-attributes
     _raw_temperature = ROUnaryStruct(_LPS2X_TEMP_OUT_L, "<h")
     _raw_pressure = ROBits(24, _LPS2X_PRESS_OUT_XL, 0, 3)
 
-    def __init__(self, i2c_bus, address=_LPS2X_DEFAULT_ADDRESS, chip_id=None):
+    def __init__(
+        self,
+        i2c_bus: I2C,
+        address: Literal[0x5C, 0x5D] = _LPS2X_DEFAULT_ADDRESS,
+        chip_id: Optional[Literal[0xB1, 0xBD]] = None,
+    ) -> None:
         self.i2c_device = i2cdevice.I2CDevice(i2c_bus, address)
         if not self._chip_id in [chip_id]:
             raise RuntimeError(
-                "Failed to find LPS2X! Found chip ID 0x%x" % self._chip_id
+                f"Failed to find LPS2X! Found chip ID {hex(self._chip_id)}"
             )
```

Comment:
Again, it makes more sense to all all `int` for `chip_id`, since we want an that type but don't need to restrict here in the base class.  Also it's interesting the way this works, it would probably be better off not providing a default argument, but it would be inconsistent with the way other drivers are instanced, so it's given the default value of `None`.  The best way to do this would be to also guarantee it is set, because if it isn't set, we'll get a `TypeError` when we try to do `hex(None)`.  So the real typing should just be `int` but technically we do feed it `None`...

So, what makes the most sense is to use another `int` as an argument to check if it was set.  `-1` seems appropriate since it's an `int` that could never be an address.  Then we can also just check for it in the guard statement.  All together, let's change the code here to be:

```suggestion
        chip_id: int = -1,
    ) -> None:
        if chip_id == -1:
            raise ValueError("Must set the chip_id argument")
        self.i2c_device = i2cdevice.I2CDevice(i2c_bus, address)
        if not self._chip_id in [chip_id]:
            raise RuntimeError(
                f"Failed to find LPS2X! Found chip ID {hex()}"
            )
```